internal/connect/command_custom_plugin_create.go

```diff
@@ -107,10 +107,6 @@ func (c *customPluginCommand) createCustomPlugin(cmd *cobra.Command, args []stri
 		if err := utils.UploadFileToAzureBlob(resp.GetUploadUrl(), pluginFileName, strings.ToLower(resp.GetContentFormat())); err != nil {
 			return err
 		}
-	} else if cloud == "GCP" {
-		if err := utils.UploadFileToGoogleCloudStorage(resp.GetUploadUrl(), pluginFileName, strings.ToLower(resp.GetContentFormat())); err != nil {
-			return err
-		}
```

Comment:
Is this functionality that currently works?

In other words, is this a breaking change to remove this?

Reply:
This was unintended change. Reverted this.. thanks for pointing it out